process-compose-mp.yml

```diff
@@ -0,0 +1,50 @@
+version: "3"
+
+environment:
+  - ESPRESSO_SEQUENCER_GENESIS_FILE=data/genesis/demo-marketplace.toml
+  - ESPRESSO_BUILDER_GENESIS_FILE=data/genesis/demo-marketplace.toml
+processes:
+  submit-transactions-public:
+    command: submit-transactions
+    environment:
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MIN_NAMESPACE=1
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MAX_NAMESPACE=6
+
+  submit-transactions-private:
+    command: submit-transactions
+    environment:
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MIN_NAMESPACE=1
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MAX_NAMESPACE=3
+
+  submit-transactions-private-fallback:
```

Comment:
I am thinking that if we also need another load generator for reserve builder? so we the submit-transactions-private load generator is for legacy one

Reply:
I think something like 8e65d4a492e0930336bbce15264cdcfc33403a86. I'm not sure of the namespaces though.

Reply:
all the private load generators are currently failing, so looking into it. Maybe there's a wrong port somewhere: 

```
│2024-09-03T20:33:25.558719Z ERROR submit_transactions: failed to submit     ││
││batch of 3 transactions err=Custom { message: "error sending request for    ││
││url (http://localhost:31003/txn_submit/batch): client error (Connect)",     ││
││status: StatusCode(500) }  
```